kyuubi-server/src/test/scala/org/apache/kyuubi/operation/KyuubiOperationPerUserSuite.scala

```diff
@@ -202,7 +202,11 @@ class KyuubiOperationPerUserSuite
         assert(executeStmtResp.getStatus.getErrorMessage.contains(
           "java.net.SocketException: Connection reset") ||
           executeStmtResp.getStatus.getErrorMessage.contains(
-            "Caused by: java.net.SocketException: Broken pipe (Write failed)"))
+            "Caused by: java.net.SocketException: Broken pipe (Write failed)") ||
+          executeStmtResp.getStatus.getErrorMessage.contains(
+            "java.net.SocketException: Socket closed") ||
+          executeStmtResp.getStatus.getErrorMessage.contains(
```

Comment:
executeStmtResp.getStatus.getErrorMessage.contains("connection does not exist")

Reply:
Thank you for your comment.
I am checking this test failure:
https://github.com/apache/kyuubi/actions/runs/4509822725/jobs/7940036880?pr=4480

It has an obviously distinct error message.
```
  "org.apache.kyuubi.KyuubiSQLException: Error operating ExecuteStatement: org.apache.thrift.transport.TTransportException: SASL authentication not complete
  	at org.apache.thrift.transport.TSaslTransport.write(TSaslTransport.java:474)
  	at org.apache.thrift.transport.TSaslClientTransport.write(TSaslClientTransport.java:37)
  	at org.apache.thrift.protocol.TBinaryProtocol.writeI32(TBinaryProtocol.java:178)
  	at org.apache.thrift.protocol.TBinaryProtocol.writeMessageBegin(TBinaryProtocol.java:106)
  	at org.apache.thrift.TServiceClient.sendBase(TServiceClient.java:70)
  	at org.apache.thrift.TServiceClient.sendBase(TServiceClient.java:62)
  	at org.apache.hive.service.rpc.thrift.TCLIService$Client.send_GetOperationStatus(TCLIService.java:469)
  	at org.apache.hive.service.rpc.thrift.TCLIService$Client.GetOperationStatus(TCLIService.java:461)
  	at org.apache.kyuubi.client.KyuubiSyncThriftClient.$anonfun$getOperationStatus$1(KyuubiSyncThriftClient.scala:384)
  	at org.apache.kyuubi.client.KyuubiSyncThriftClient.$anonfun$withLockAcquiredAsyncRequest$2(KyuubiSyncThriftClient.scala:155)
  	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
  	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
  	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
  	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
  	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
  	at java.lang.Thread.run(Thread.java:750)
```
Do you have any ideas?

Reply:
@pan3793 
Yeah, that's a great idea.